### PR TITLE
Add physical_validation recipe

### PR DESCRIPTION
**Pull request by mattwthompson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/physical_validation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pymbar
+
+#test:
```
Comment (synapticarbors) [inline recipes/physical_validation/meta.yaml:28]: Please remove these comments

File: recipes/physical_validation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: https://github.com/shirtsgroup/physical_validation/archive/{{ version }}.tar.gz
+  sha256: 8b7b80bbe22e555a290283e0eaa1edd6a20440f7f5e5f96887d7a51ac3c64dc5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```
Comment (synapticarbors) [inline recipes/physical_validation/meta.yaml:16]: conda-forge is no longer building packages based on python 2.7 so you can remove the `skip: True` statement

File: recipes/physical_validation/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% set name = "physical_validation" %}
+{% set version = "1.0rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shirtsgroup/physical_validation/archive/{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/physical_validation/meta.yaml:9]: I believe the url should be:

```
https://github.com/shirtsgroup/physical_validation/archive/v{{ version }}.tar.gz
```

note the missing `v` before the version templating.

File: recipes/physical_validation/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#test:
+#  # Some packages might need a `test/commands` key to check CLI.
+#  # List all the packages/modules that `run_test.py` imports.
+#  imports:
+#    - simplejson
+#    - simplejson.tests
+
+about:
+  home: https://github.com/shirtsgroup/physical_validation
+  license: LGPL-2.1
```
Comment (synapticarbors) [inline recipes/physical_validation/meta.yaml:37]: To satisfy the spdx descriptors you need either `LGPL-2.1-only` or `LGPL-2.1-or-later`. See:

https://spdx.org/licenses/

**Comment by mattwthompson** [comment]
Indeed, this has gone stale. Will re-open if we pick it back up

**Pull request closed without merging**